benefits/static/css/styles.css

```diff
@@ -245,46 +253,67 @@ footer .footer-links li a:visited {
 }
 
 /* Buttons */
+/* Primary Button: Use all three classes: btn btn-lg btn-primary */
+/* Set button width in parent with Bootstrap column */
+/* Height: 60px on Desktop; 72 on mobile*/
 
-.btn-lg {
-  border-width: 2px;
+:root {
+  --primary-button-padding: 19px 0;
 }
 
-.btn-link {
-  color: var(--primary-color);
+@media (min-width: 992px) {
+  :root {
+    --primary-button-padding: 13px 0;
+  }
 }
 
-.btn-primary {
+.btn-lg.btn-primary.btn-lg {
```

Comment:
Do we need the double `.btn-lg` selector here?

Reply:
removed. replaced the extra`btn-lg` with `btn`. I'm keeping it like this with all 3 classes for now until the Button refactor ticket, b/c, right now, all the buttons have all 3 of these classes.